content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -106,3 +106,54 @@ Once that option is enabled, go back to the :guilabel:`Profile` options page, an
 
 Once all those steps have been taken, perform a test scan to ensure the Zebra scanner is working
 properly, as intended.
+
+Honeywell mobile computer scanner
+=================================
+
+When using Honeywell scanners, ensure the following configurations are set to prevent errors.
```

Comment:
I asked @mejg-odoo for extra context on why we need these configurations to prevent errors. Feel free to modify my suggestion as you see fit!
```suggestion
When using Honeywell scanners, change the *wedge method* settings to *Keyboard* so that the device
can process barcodes as keyboard characters and symbols.
.```

Reply:
That logic makes sense to me! Suggestion implemented ✅